### PR TITLE
Implement generate bitly url JS

### PR DESCRIPTION
**Pull request by arshxyz**
Demo function to generate bit.ly shortened URL in JS

References #82

### CODE REVIEW

File: nodejs/generate-bitly-url/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+// Bitly access token (can be fetched from https://app.bitly.com/settings/api/)
+const token = process.env.BITLY_ACCESS_TOKEN;
+const config = {
+    headers: {
+        'Authorization': `Bearer ${token}`,
+    }
+};
+
+// Getting the URL from Appwrite's environment variable
+const payload = JSON.parse(process.env.APPWRITE_FUNCTION_DATA);
```
Comment (Meldiron) [inline nodejs/generate-bitly-url/index.js:13]: No reason to use JSON as we only get 1 input and it's `url`. The input can simply be `string` holding our `url`. Please also document this in `README.md` so users know they need to provide this input and how.

File: nodejs/generate-bitly-url/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "long_url": payload['url']
+};
+
+// Make a request to bitly and output the shortened URL
+axios.post('https://api-ssl.bitly.com/v4/shorten', 
+  data,
+  config,
+  )
+  .then(function (response) {
+    console.log(response.data.link);
```
Comment (Meldiron) [inline nodejs/generate-bitly-url/index.js:24]: Please document this in `README.md`so users know what to expect.

File: nodejs/generate-bitly-url/index.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  })
+  .catch(function (error) {
+    console.log(error);
```
Comment (Meldiron) [inline nodejs/generate-bitly-url/index.js:27]: If we get an error, we need to use `console.error()` so the error is sent in the `stderr` and we can find out if the function was a success or not when fetching function status on frontend.

**Review by Meldiron — CHANGES_REQUESTED** [review]
Great job, we just need to do a few small changes 😊

**Comment by arshxyz** [comment]
@Meldiron Hi, I've made the required changes. Thanks for the review!

**Review by Meldiron — APPROVED** [review]
Looks good to me!

**Comment by Meldiron** [comment]
THANK YOU! All changes merged 🥳

Please reach out to `eldadfux` on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+is%3Aissue+label%3Ahacktoberfest+created%3A%3E2021-01-01+is%3Aopen&type=issues) and help make Appwrite better :)

**Pull request merged**